Core/Core/Models/Base.cs

```diff
@@ -276,6 +276,20 @@ public virtual string speckle_type
         return __type;
       }
     }
+
+    private string __simpleType;
+
+    [JsonIgnore]
+    public string simple_Type
```

Comment:
I would prefer not to add this as a property to `Base`

Calling `myObject.GetType().Name` in the report seems fine, no?

Adding new properties to `Base` requires a fair bit of commitment from us to maintain them, ensure don't break, unit test, align across SDKs, and document.
And in this case, I don't think this property brings enough value to justify that.

Reply:
ok! I got it! I changed the report call.